api/v1alpha08/sonataflow_types.go

```diff
@@ -179,6 +186,11 @@ func (s *SonataFlowStatus) IsBuildFailed() bool {
 	return cond.IsFalse() && cond.Reason == api.BuildFailedReason
 }
 
+func (s *SonataFlowStatus) IsWaitingForChanges() bool {
```

Comment:
I'll rename the method accordingly to the reason and so `IsWaitingConfigurationChanges`

Reply:
fixed